android/templates/app/webpack-default/template/platform/android/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file something that may get generated/overridden by the `semantic.colors.json` file for light/dark themes?

Reply:
AFAIK we generate a `ti.semantic.colors.xml` for that, so we don't touch the default `colors.xml`.